server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -173,6 +186,11 @@ export class DeliLambda extends EventEmitter implements IPartitionLambda {
 
     public handler(rawMessage: IQueuedMessage) {
         let kafkaCheckpointMessage: IQueuedMessage | undefined;
+        const lumberJackMetric = DefaultServiceConfiguration.enableLumberTelemetryFramework ?
```

Comment:
Is the idea to keep this check here forever? Or just temporarily while we do performance analysis? I'm wondering if we would need to add that check everywhere we create a lumber metric. Also, Lumberjack's logging feature would still be executed if we don't set `enableLumberTelemetryFramework` at the Lumberjack level.

@GaryWilber @pradeepvairamani any thoughts?

Reply:
I think it makes sense to have this check until we are confident that the performance impact is minimal. For the logs, I was planning to put it behind this or a different flag in a separate PR since this PR is only using metrics and it is kind of big already. But thanks for calling that out.

Reply:
I would hope temporarily while we do performance analysis. 

In general we should be mindful around lambda changes due to the raw amount of messages they are processing per second. Because of that, small changes could produce large performance changes - I don't mean to say lumberjack would cause such a thing, just bringing it up because it's important :)

At some point we should get some performance tests that run in the pull request flows that could automatically detect regressions in the ops/second for the lambdas (future FHL project idea?)